generators/app/templates/src/containers/TicTacToe/GameListPage/children/Form/game-form.component.js

```diff
@@ -72,7 +72,19 @@ const GameForm = ({ webId, sendNotification, opponent, setOpponent }: Props) =>
        * */
       if (inboxes.length > 0) {
         const newDocument = await ldflexHelper.createNonExistentDocument(documentUri);
-        if (newDocument) {
+
+        /**
+         * If game already exist show an error message
+         */
+        if (!newDocument) {
+          errorToaster(`${documentUri} ${t('game.alreadyExists')}`, t('notifications.error'));
+          return null;
+        }
+
+        /**
+         * if document was created we will initial the game if not show an error
```

Comment:
Should read "If document was created we will initialize the game, otherwise show an error"

Reply:
done